odata-csdl/14 Vocabulary and Annotation.md

```diff
@@ -1699,12 +1699,9 @@ types](#BuiltInTypesfordefiningVocabularyTerms)
 `Edm.NavigationPropertyPath`, `Edm.AnyPropertyPath`, or `Edm.ModelElementPath`.
 Its argument is a [model path](#PathExpressions) with the following
 restriction:
-- A non-null path MUST resolve to a model
-element whose type is an entity type, or a collection of entity types,
-e.g. a navigation property.
+- A non-null path MUST resolve to a [navigation property](#NavigationProperty).
```

Comment:
Would we expect a NavigationPropertyPath to resolve to an entity set or a singleton?

Definitely not in the SAP vocabularies...